src/main/webapp/i18n/de/exam.json

```
@@ -313,6 +313,12 @@
                 "endDate": "Ende Bearbeitungsfenster",
                 "workingTimeTooltip": "Die erlaubte Bearbeitungszeit je Durchlauf"
             },
+            "monitoring": {
+                "enable": {
+                    "title": "Aktiviere die Klausur Überwachung",
```

just a suggestion 🤷‍♂️ 
```suggestion
                    "title": "Klausurüberwachung aktivieren",
```